test/connectivity/modules/NetworkController/NetworkProfileSetting.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace NetworkController
+{
+    public class NetworkProfileSetting
+    {
+        public int Delay { get; set; }
```

Comment:
should these parameters use uint type?  And are all validations done in Settings? for example PackageLoss should be between 0 and 100.

Reply:
updated to uint and added validation for packageloss. Also added bandwidthunit to be able to validate the value.